iphone/metadata/fr-FR/release_notes.txt

```diff
@@ -1,13 +1,25 @@
-• Nouvelles données OpenStreetMap au 27 février
-• Correction d'un bug sur l'iPad lorsqu'un clavier est connecté
-• Correction d'un double-clic inutile sur le bouton de l'itinéraire
-• Suppression des étages et des cellules de code postal dupliqués
-• Activation des notes de l'éditeur OpenStreetMap pour tous les lieux sélectionnés
-• Recherche corrigée et améliorée
-• Séparation des cuisines végétaliennes et végétariennes dans la recherche
-• Correction des styles de carte pour les aéroports, terrains, voies navigables et zones hachurées.
-• Ajout des spécialistes de soins de santé: alternative, audiologist, blood_donation, optometrist, podiatrist, psychotherapist, sample_collection, speech_therapist
-• Ajout des tours de communication, des générateurs solaires, éoliens, gaziers et hydroélectriques
-• Amélioration des traductions en arabe, norvégien, portugais et turc.
+• New OSM Planet data as of March 26
+• Add localized links for the OSM wiki page
+• Replaced uphill and downhill icons with more clear arrows
+• Correctly import KML bookmarks with track data
+• Disable 3D buildings preference if power saving is enabled
+• Music and music instruments shops can be added using Editor
+• Sewing shops and haberdasheries are searchable and have a better icon
+• Searching for a "supermarket" doesn't result in convenience stores anymore
 
-Suivez nos actualités sur : omaps.app/news
+Styles
+• New shop types: agrarian, appliance, bed, outpost, gas, key cutter, locksmith, carpet, motorcycle repair, pet grooming, hi-fi, bathroom furnishing, curtain, fashion accessories, cannabis, herbalist
+• Added money transfer, bicycle charging stations, power plant
+• Made captions optional for most of POI icons
+• Made park, forest, and scrub colors different
+• Added metal construction icons
+• Improved aerial ways, cable cars, power lines, fences
+• Reduce Barcelona subway visibility
+• Fixed isolines that are drawn over buildings, roads, etc.
+• Fixed rendering of multiple-type features (e.g. some POI captions were displayed too early)
+
+Translations:
+• Improved German, Turkish, Belarusian, Hungarian translations
+• Improved Arabic TTS instructions
+
+Follow our news at omaps.app/news
```

Comment:
* Nouvelles données Openstreetmap du 26 Mars
* Ajout de liens localisés pour la page wiki OSM
* Remplacement des icônes de montée et de descente par des flèches plus claires
* Importation correcte des signets KML avec les données de la trace
* Désactivation des préférences pour les bâtiments en 3D si l'économie d'énergie est activée
* Les magasins de musique et d'instruments de musique peuvent être ajoutés à l'aide de l'éditeur
* Les magasins de couture et les merceries peuvent être recherchés et ont une meilleure icône
* La recherche d'un "supermarché" n'aboutit plus à des supérettes.

Styles
* Nouveaux types de magasins : agricole, appareils électroménagers, literie, point de retrait, gaz, reproduction de clés, serrurier, tapis, réparation de motos, toilettage pour animaux, hi-fi, ameublement de salle de bain, rideaux, accessoires de mode, cannabis, herboristerie
* Ajout du transfert d'argent, des stations de recharge de vélos, des centrales électriques
* Légendes optionnelles pour la plupart des icônes de POI
* Différenciation des couleurs des parcs, des forêts et des broussailles
* Ajout d'icônes de construction en métal
* Amélioration des voies aériennes, des téléphériques, des lignes électriques, des clôtures
* Réduction de la visibilité du métro de Barcelone
* Correction des courbes de niveaux dessinées sur les bâtiments, les routes, etc
* Correction du rendu des éléments de type multiple (par exemple, certaines légendes de POI s'affichaient trop tôt)

Traductions :
* Amélioration des traductions en allemand, turc, biélorusse et hongrois
* Amélioration des instructions TTS en arabe